src/emu.rs

```diff
@@ -249,7 +256,8 @@ impl Emu {
             self.maps.is_64bits = true;
             self.init_regs_dts9_patcherv();
```

Comment:
if you want to put these both behind an if flag, would be cool for others (if any)